packages/block-library/src/image/image-editor.js

```diff
@@ -60,12 +60,19 @@ function AspectGroup( { aspectRatios, isDisabled, label, onClick, value } ) {
 	);
 }
 
-function AspectMenu( { isDisabled, onClick, value, defaultValue } ) {
+function AspectMenu( {
+	toggleProps,
+	isDisabled,
+	onClick,
+	value,
+	defaultValue,
+} ) {
 	return (
 		<DropdownMenu
 			icon={ aspectRatioIcon }
 			label={ __( 'Aspect Ratio' ) }
 			popoverProps={ POPOVER_PROPS }
+			toggleProps={ toggleProps }
```

Comment:
`toggleProps` wasn't being passed to this component. That's why it wasn't being considered a `ToolbarItem`.